design-book/src/vision.md

```diff
@@ -0,0 +1,24 @@
+# Vision
```

Comment:
Largely for the editor as a whole. It will change as we get user feedback, but I'm really not seeing any appetite for a do-everything application for making games from Bevy's users or prospective users.

Reply:
It's hard to discuss the latter without the former. We need a base on which to build, and can re-evaluate larger scopes once we have more basic functionality working.

Unity/godot/unreal didn't get where they are today by aiming for the stars. They built something minimal (like hammer) and slowly evolved into something more.